UKHomeOffice/cerberus-service
# Cop-8935 - Hide unnecessary sections in task details for different mode types

## Description
This PR hides unnecessary sections in task details based on different mode types.
Jira Ticket: [https://support.cop.homeoffice.gov.uk/browse/COP-8935](url)

## To Test
- Pull and run against dev
- When viewing any RoRo Unaccompanied task details, verify that the following section is hidden: **Vehicle details**.
- When viewing any RoRo Tourist task details, verify that the following sections are hidden: **Goods**, **Account details** & **Haulier details**.


## Developer Checklist

\* Required
